test/test_play.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of alert-stream-simulator.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import unittest
+
+import pytest
+import astropy.time
+
+from streamsim import creator, player, _kafka
+from test import testutils
+
+
+@pytest.mark.integration_test
+class TestPlayIntegration(unittest.TestCase):
+    @staticmethod
+    def mock_alert(alert_id, timestamp):
```

Comment:
Do you want to use your utility code again here?

Reply:
Looking at it more, is this even used anymore?

Reply:
Oops, you're right. I added the utility code but forgot to remove this spot, which is where I first wrote it.